examples/updatecli.d/sources/dockerfile_sources.yaml

```
@@ -23,9 +23,9 @@ sources:
         kind: latest
 conditions:
   isENVSet:
-    sourceid: helm
-    name: Is the 2nd ENV instruction having a "keyword" set to "HELM_VERSION"
-    kind: dockerfile
+    sourceid: "helm1"
+    name: 'Is the 2nd ENV instruction having a "keyword" set to "HELM_VERSION"'
+    kind: "dockerfile"
```

```suggestion
    sourceid: helm1
    name: 'Is the 2nd ENV instruction having a "keyword" set to "HELM_VERSION"'
    kind: dockerfile
```

otherwise use the same quoting everywhere instead to be coherent ;)

---

That's true, I'll finish the cleanup :-)

---

I added quotes to all manifest examples under the examples directory. But I didn't fix the content of those have been unmaintained for a quite long time so maybe we should remove them at some point

---

they were just useful to test that `updatecli manifest upgrade` can update them to the latest syntax